.circleci/config.yml

```diff
@@ -1,58 +1,83 @@
-version: 2 # use CircleCI 2.0
+version: 2
+
+.my_job: &myjob
+  working_directory: ~/liboqs-java
+  docker:
+    - image: ${IMAGE}
+  steps:
+    - checkout
+    - run:
+        name: Install liboqs dependencies
+        command: ${SUDO} apt-get update && ${SUDO} apt-get install cmake gcc ninja-build libssl-dev python3-pytest python3-pytest-xdist unzip xsltproc doxygen graphviz -y
+    - run:
+        name: Clone liboqs-master
+        command: .circleci/git_no_checkin_in_last_day.sh || (cd ~/ && git clone --branch master --single-branch https://github.com/open-quantum-safe/liboqs)

```

Comment:
This looks like a hen-and-egg problem: You need the liboqs code locally before the (new) `git_no_checkin_in_last_day.sh` can run OK --> "Clone liboqs-master" should unconditionally git clone liboqs.
Except this one, the `git_no_checkin_in_last_day.sh` guard must be before each run step we don't really want to execute if nothing changed. Right now it only guards build liboqs - but the other run steps also wouldn't be necessary.

Reply:
Oh, right, I see. Just to clarify, I should remove `git_no_checkin_in_last_day.sh` before building liboqs and add to every other step (including the `Install liboqs dependencies` step)?

Reply:
@baentsch, I'll let you do the merge when you feel this is ready.

Reply:
> Oh, right, I see. Just to clarify, I should remove git_no_checkin_in_last_day.sh before building liboqs 

Yes

> and add to every other step 

Yes: Only this will ensure that no further work is performed without a need to do so (but the test still passes "green")

> (including the Install liboqs dependencies step)?

No: That wouldn't be sensible as at that time liboqs has not been locally cloned for the script to work OK.

Reply:
> @baentsch, I'll let you do the merge when you feel this is ready.

@christianpaquin Sure, I'd do -- but I think I don't have write access to _this_ repo: my approving review didn't change the PR status. So I probably won't be permitted to merge either.